desk/lib/bolt.hoon

```diff
@@ -6,24 +6,23 @@
 ::  ...
 ::  %-(agent:bolt your-agent)
 ::
-
 /-  *bolt
 /+  verb, agentio
-
+::
 =>
-|% 
-+$  card  card:agent:gall
-+$  versioned-state
-  $%  state-0
-  ==
-+$  state-0  
-  $:  %0 
-      which=?
-      kids=?
-      =blacklist 
-      =whitelist
-  ==
---
+  |% 
+  +$  card  card:agent:gall
+  +$  versioned-state
+    $%  state-0
+    ==
+  +$  state-0  
+    $:  %0 
+        white=?
```

Comment:
Since this is no longer "should we use blacklist or whitelist" but rather "should we use whitelist" I suggest that you change this. Makes the condition further down clearer too.